test/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM centos:7
+ADD . /bin
+ENTRYPOINT [ "/bin/oauth-proxy" ]
+ENV OPENSHIFT_BUILD_NAME oauth-proxy-5
```

Comment:
?

Reply:
an artifact from copying the Dockerfile. and my update to vendor pulled in everything, I'll fix that.